### ISSUE TITLE
[Suggestion] Support for equips slots from "The Aether" mod?

### ISSUE THREAD

**Issue by SonicX8000** [Laike-Endaril/Set-Bonus#25]
Mod in question:
https://www.curseforge.com/minecraft/mc-mods/the-aether

I wonder if it's possible to apply 'set bonus' effects to the items that The Aether adds, like for the capes, gloves, necklaces, shields and the misc. accessories.

There's currently 2 Ring Slots, 2 Misc. Accessories, a Necklace Slot, a Shield Slot, a Gloves Slot, and a Cape Slot. One idea I wanted to do is give the gloves some sort of boost since they don't have any effects on their own other than being used for set effects with some armor from The Aether.

**Comment by TheGameCreator13** [comment]
Well for the mod **Baubles** You can define Rings for example by typing `bauble_ring` to define that slot as the requirement. I would suggest looking into what **The Aether** Mod has its slots defined as. It may be compatible with SetBonus' system

**Comment by Laike-Endaril** [comment]
Set Bonus only has explicit slot keywords defined for vanilla slots and Baubles slots, but depending on how Aether's slots are implemented, you may be able to refer to them by raw slot number.

The vanilla offhand slot is 40, which is the last vanilla slot iirc, so if they added their pushed their slots into the vanilla inventory itself, they would probably be above 40

**Comment by SonicX8000** [comment]
Late reply, but checking the Aether slots... it might be using their own slots since it uses it's own inventory for their equipment.

**Comment by SonicX8000** [comment]
I happen to see that this mod recently got updated for it's 1.12.2 version, would it be possible to add in support for the Accessory Slots that the Aether Mod adds?

Trinkets & Baubles could be neat to mess with too, as that mod has it's own Trinket GUI Slots to use if config'd... though I dunno how that would work since you can change how many slots you can have at once.

This is the Trinkets & Baubles Mod:
https://legacy.curseforge.com/minecraft/mc-mods/trinkets-and-baubles

**Comment by Laike-Endaril** [comment]
Yep, I'm back for now.  I can give it a look and see what I find.

**Comment by Laike-Endaril** [comment]
Ah...ignore that bit that says
"For #25, superior JEI compat performance..."
I reference the wrong issue number there.  It should've said for #27

**Comment by Laike-Endaril** [comment]
These features are now implemented in the latest release (1.12.2.029)
